evap/static/scss/components/_range-slider.scss

```
@@ -0,0 +1,43 @@
+section.range-slider {
+    position: relative;
+    height: 45px;
+    text-align: center;
+}
+
+section.range-slider input {
+    pointer-events: none;
+    position: absolute;
+    left: 0;
+    top: 20px;
+}
+
+section.range-slider input::-webkit-slider-thumb {
+    pointer-events: all;
+    position: relative;
+    z-index: 1;
+    outline: 0;
+}
+
+section.range-slider input::-moz-range-thumb {
+    pointer-events: all;
+    position: relative;
+    z-index: 10;
+    -moz-appearance: none;
+}
+
```

correct color for chrome:
```suggestion
section.range-slider input::-webkit-slider-runnable-track {
    background-color: $light-gray;
}

```